code/modules/hydroponics/growninedible.dm

```diff
@@ -59,3 +60,10 @@
 /obj/item/grown/on_grind()
 	for(var/i in 1 to grind_results.len)
 		grind_results[grind_results[i]] = round(seed.potency)
+
+/obj/item/grown/on_brew()
+	var/list/my_reagents = list()
+	for(var/R in reagents.reagent_list)
+		var/datum/reagent/RE = R
+		my_reagents += RE.type

```

Comment:
IIRC, there is an existing reagents proc to do this automatically.

Reply:
i dont know the name of said proc